_pages/home/updating-to-version-2211.35/migration.md

```diff
@@ -66,7 +66,22 @@ The update to Spartacus 2211.36 is mostly focused on updating the framework to A
 
 **Note:** You must start with a version 2211.32.1 Spartacus app to be able to update to version 2211.36.
 
-1. To update to version 2211.36 of Spartacus, run the following command in the workspace of your Angular application:
+1. In `package.json`, update the version to 2211.36.1 for all Spartacus dependencies except `@spartacus/schematics`.
+
+1. In `package.json`, if you haven't already done so, change the version range specifier for `@spartacus/schematics` from `^2211.32.1` to `~2211.32.1`.
+
+   **Note:** For `@spartacus/schematics` in `package.json`, you are replacing `^` with `~`, but the version (`2211.32.1`) stays the same. It should **not** be changed to `2211.36.1`.
```

Comment:
IMHO it will be more natural/logical if we swap those 2 first numbered points:
1. First let's make sure the range specifier for `@spartacus/schematics` is `^` instead of `~` in `package.json`
2. Only later set all `@spartacus` libs except schematics to "~2211.36.1" in `package.json`

Reply:
Sounds good! 👍